From 4278acc981944922e44ad48357d2b8f1fe2d33b6 Mon Sep 17 00:00:00 2001
From: Yiqun Lin <yqlin@apache.org>
Date: Tue, 26 Sep 2017 17:21:42 +0800
Subject: [PATCH 2691/2748] HDFS-12495.
 TestPendingInvalidateBlock#testPendingDeleteUnknownBlocks
 fails intermittently. Contributed by Eric Badger.

(cherry picked from commit e9b790db9de7c3fbe451bcdc3fd928a27a0d8b4e)
(cherry picked from commit fb5de93c466120908e2c9f2d63dc954395c487e4)

Change-Id: I2c0766cfe4e6bc625cfa0794414086e731fec178
Ref: CDH-61949
---
 .../TestPendingInvalidateBlock.java                |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestPendingInvalidateBlock.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestPendingInvalidateBlock.java
index 84ae771..cb2d8f5 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestPendingInvalidateBlock.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestPendingInvalidateBlock.java
@@ -140,7 +140,7 @@ public void testPendingDeleteUnknownBlocks() throws Exception {
     Assert.assertEquals(0L, cluster.getNamesystem().getPendingDeletionBlocks());
     // restart DataNodes
     for (int i = 0; i < REPLICATION; i++) {
-      cluster.restartDataNode(dnprops[i], true);
+      cluster.restartDataNode(dnprops[i]);
     }
     cluster.waitActive();
 
-- 
1.7.9.5

